### ISSUE TITLE
Request for training

### ISSUE THREAD

**Issue by sehooni** [lhatsk/AlphaLink#17]
Hello, thank you for great research.

I already install alphalink in my workspace following by openfold page.
Actually, I want to try to train the model like in your paper.
So could you provide any training script which I follow?

In the paper, you guys train the model with fine-tuning method.
And I want to follow up your script.
So if there are any scripts to follow, please provide to us.

Thank you for reading.

**Comment by lhatsk** [comment]
I don't quite remember why I didn't take the full set. There are two possibilities that come to mind: I think I originally had a train/ validation split on the trRosetta training data from a previous project and only used the training split. Another possibility is that I didn't have crosslinks for all the targets. Just take as much as you can. You would need to simulate the photoAA crosslinking data. I did it on the fly during training.

> Additionally, I want to ask how to consist of the data with crop size!
I used the default crop size for fine-tuning which is 384AA.

**Comment by sehooni** [comment]
thank you for explaining for my question.

Additionally, I have one more question.
In the process of training, I have a multi-gpus, and I want to try to use all of it.

By the way, there are original code, openfold, there are some args to set gpus, so I do it just setting gpus as 2.
But there was some error, which are came from the setting the output gpu because they have to combine the output data.

So I want to ask some advices to train with multiple gpus.

**Comment by lhatsk** [comment]
I didn't alter the cropping strategy. It's a random crop of size 384AA.

**Comment by sehooni** [comment]
hello! I want to ask something.

I can check the Neff in inference process.
However, is there any setting to train with neff? 
Or, does the model train itself about neff?

thank u for reading.

**Comment by lhatsk** [comment]
No, there is no setting. You would just need to replicate what is being done during inference and subsample the MSAs beforehand.

**Comment by sehooni** [comment]
Then in train process, does it use all msa to train it and does neff setting use only in inference process to set the msa condition??

**Comment by lhatsk** [comment]
Actually, in case the features are preprocessed, MSA subsampling is on by default during training: https://github.com/lhatsk/AlphaLink/blob/main/openfold/data/data_modules.py#L259 If you move it outside the if, it will work regardless.
